website/docs/02_components/display/_category_.yml

```
@@ -0,0 +1,3 @@
+label: Layout
```

My only blocking change request!
```suggestion
label: Display
```